src/helpers/getStatusBarText/index.ts

```diff
@@ -1,11 +1,24 @@
+import { workspace, WorkspaceConfiguration } from 'vscode'
 import getFormattingStatus from '../getFormattingStatus'
 
-export const ENABLED_TEXT = 'Formatting: $(check)'
-export const DISABLED_TEXT = 'Formatting: $(x)'
+type StatusBarLabelT = {
+  formattingEnabled: string
+  formattingDisabled: string
+};
 
-const getStatusBarText = () => {
+export const getStatusBarText = (): string => {
   const isFormattingActivated = getFormattingStatus()
 
+  const configuration: WorkspaceConfiguration = workspace.getConfiguration();
+  let statusBarLabel: StatusBarLabelT | undefined
+
+  statusBarLabel = configuration.get<StatusBarLabelT>(
+    'formattingToggle.statusBarLabel'
+  );
+
+  const ENABLED_TEXT = statusBarLabel!.formattingEnabled
+  const DISABLED_TEXT = statusBarLabel!.formattingDisabled
```

Comment:
It’s really a constant anymore!
```suggestion
  const {formattingEnabled, formttingDisabled} = statusBarLabel
```